test/unit/test_application.js

```diff
@@ -72,6 +72,33 @@ var runApp = function() {
   });
 };
 
+exports['test instantiate the user module with the required parameters'] = function(done) {
```

Comment:
Can you switch to the describe API for defining your tests. That's the convention we use for mocha.

See https://github.com/feedhenry-raincatcher/raincatcher-user/pull/19/files#diff-45a64b8b3cef7d7a7b79beb5d94db350R10 as an example